core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -576,10 +575,13 @@ export class Topic {
     }
     let diagnosticTestSkillSummaries = this.getDiagnosticTestSkillSummaries();
 
+    const skillIdToDiagnosticTestMap = new Map();
```

Comment:
There might be something to be said for just doing this using a regular dict, i.e. {}, and using hasOwnProperty() for the check.

I mention this because we once were trying to help get Oppia working on some computers in Ghana and they were using really old browsers, so stuff like includes() etc. didn't work.

Reply:
Thank you, I have updated this with regular dict.